xml/Menu/birthdays.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<menu>
+  <item>
+    <path>civicrm/admin/birthdays</path>
+    <page_callback>CRM_Birthdays_Form_Settings</page_callback>
+    <title>Birthdays Extension Settings</title>
+    <access_arguments>administer CiviCRM</access_arguments>
+    <adminGroup>Communications</adminGroup>
+    <desc>Configure template and outgoing email address for automatic birthday email notifications</desc>
+  </item>
```

Comment:
Let's add a navigation menu item and an icon. See https://github.com/systopia/de.systopia.donrec/blob/2d4fae2241f99404ae1a384a6180f47c38522fc1/xml/Menu/donrec.xml#L33-L43 and https://github.com/systopia/de.systopia.donrec/blob/2d4fae2241f99404ae1a384a6180f47c38522fc1/donrec.php#L316-L325 for an example.

Reply:
I don't fully understand the point of linking using the settings page as a menu item as ideally it's only configures onse